linters/cmake-format/plugin.yaml

```diff
@@ -0,0 +1,30 @@
+version: 0.1
+tools:
+  definitions:
+    - name: cmake-format
+      runtime: python
+      package: cmake-format
+      shims: [cmake-format]
+      known_good_version: 0.6.13
+lint:
+  definitions:
+    - name: cmake-format
+      files: [cmake]
+      commands:
+        - name: lint
+          run: cmake-format --check ${target}
+          success_codes: [0, 1]
+          read_output_from: stderr
+          output: pass_fail
+        - name: fix
+          formatter: true
+          output: rewrite
+          success_codes: [0, 1]
+          run: cmake-format -i ${target}
+          in_place: true
+      tools: [cmake-format]
+      suggest_if: files_present
+      known_good_version: 0.6.13
```

Comment:
Can you also add the following:
```yaml
direct_configs:
  - cmake-format.yaml
  - cmake-format.json
  - cmake-format.py
  - .cmake-format.yaml
  - .cmake-format.json
  - .cmake-format.py
```
https://cmake-format.readthedocs.io/en/latest/configuration.html